[NNC] Fix llvm min lowering for int inputs

eellison opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* #47374 [NNC] Enable unary op cpu testing
* #47373 [NNC] refactor cuda half support to more general file
* #47372 [NNC] More cpu tests
* #47371 [NNC] Add more CPU Tests
* **#47370 [NNC] Fix llvm min lowering for int inputs**
* #47369 Add more CPU tests
* #46951 [TensorExpr][CPU] Fix bool -> int casting
* #46950 Small changes/cleanup
* #46949 Force LLVM Compilation for CPU Tests

Differential Revision: [D24805249](https://our.internmc.facebook.com/intern/diff/D24805249)

Commit:
Fix llvm min lowering for int inputs

[ghstack-poisoned]

Commit:
Update on "Fix llvm min lowering for int inputs"




[ghstack-poisoned]

Commit:
Update on "Fix llvm min lowering for int inputs"




[ghstack-poisoned]

Commit:
Update on "Fix llvm min lowering for int inputs"




[ghstack-poisoned]

Commit:
Update on "[NNC] Fix llvm min lowering for int inputs"




[ghstack-poisoned]

bertmaher reviewed (approved):
Sort of shocking this was already done for max and not min :-o

Commit:
Update on "[NNC] Fix llvm min lowering for int inputs"




[ghstack-poisoned]

Commit:
Update on "[NNC] Fix llvm min lowering for int inputs"


Differential Revision: [D24805249](https://our.internmc.facebook.com/intern/diff/D24805249)

[ghstack-poisoned]

Commit:
Update on "[NNC] Fix llvm min lowering for int inputs"


Differential Revision: [D24805249](https://our.internmc.facebook.com/intern/diff/D24805249)

[ghstack-poisoned]
